api/v1beta1/tags.go

```diff
@@ -119,6 +119,10 @@ const (
 	// APIServerRole describes the value for the apiserver role.
 	APIServerRole = "apiserver"
 
+	// APIServerRoleInternal describes the value for the apiserver-internal role,
+	// an identifier for an internal load balancer serving apiserver traffic for cluster nodes.
+	APIServerRoleInternal = "apiserver-internal"
```

Comment:
Is the internal load balancer required for the /etc/hosts change to work? Or is this orthogonal?

Reply:
Yes, internal load balancer is required for the /etc/hosts fix to work. 
The IP address of the load-balancer (defaulted to 10.0.0.100) is used to resolve APIServer's public DNS name via the update in `/etc/hosts`.